lib/services/ca.go

```diff
@@ -86,7 +86,7 @@ func (c *CertAuthID) Check() error {
 		return trace.Wrap(err)
 	}
 	if !cstrings.IsValidDomainName(c.DomainName) {
-		return trace.BadParameter("'%v' is a bad domain name", c.DomainName)
+		return trace.BadParameter("Identity validation error: '%v' is not a valid FQDN", c.DomainName)
```

Comment:
error sentences should be lowercase, they will be most likely included in logs:

https://github.com/golang/go/wiki/CodeReviewComments#error-strings